### ISSUE TITLE
`simple-markdown` is not listed as a peer dependency

### ISSUE THREAD

**Issue by joelpierre** [GetStream/stream-chat-react-native#1787]
**Describe the bug**
Peer dependency for `simple-markdown` not specified in any package.json.

**Dev environment info (please complete/provide the following information):**

- Package version - v5.2.0
- react-native
- Device/Emulator (android/ios) and OS version - BOTH

**To Reproduce**
Run a dev build or standard build

**Expected behavior**
You should list it as a peer dependency so we know to install it and what version

**Screenshots**

![Screenshot 2022-11-08 at 10 49 40](https://user-images.githubusercontent.com/8969851/200551656-67831aa5-3ec2-4236-a162-37ea0df0b3ab.png)

**Comment by madsroskar** [comment]
`simple-markdown` is a dependency of `react-native-markdown-package`, which in turn is a dependency of `stream-chat-react-native-core`, and should be available through your package manager when that is resolved. Are you able to make a minimal reproduction of this issue? When I set up a simple app using the SDK, I'm not able to reproduce an issue of `simple-markdown` not being available.

**Comment by joelpierre** [comment]
Yeah I noticed that too. So was surprised it wasn't "auto" resolved. We are however using a mono repo with yarn workspaces with an installConfig.hoistingLimit set to `dependencies`. Theoretically it should install all of your modules packages regardless of this setting however it is missing.

Our fix for now is just to add that as a dependency for us. However with that being said, if you use the package directly in your core package I think you should reference it in your package.json?

I can't get to minimal reproduction for now 😅
